x-pack/solutions/security/plugins/security_solution/public/flyout/entity_details/universal_right/index.tsx

```diff
@@ -7,20 +7,37 @@
 
 import React, { useEffect } from 'react';
 import type { FlyoutPanelProps } from '@kbn/expandable-flyout';
+import { METRIC_TYPE } from '@kbn/analytics';
 import {
-  uiMetricService,
   UNIVERSAL_ENTITY_FLYOUT_OPENED,
+  uiMetricService,
 } from '@kbn/cloud-security-posture-common/utils/ui_metrics';
-import { METRIC_TYPE } from '@kbn/analytics';
-import type { EntityEcs } from '@kbn/securitysolution-ecs/src/entity';
-import type { EsHitRecord } from '@kbn/discover-utils';
+import { EuiEmptyPrompt, EuiLoadingSpinner } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useGetGenericEntity } from './hooks/use_get_generic_entity';
+import { FlyoutNavigation } from '../../shared/components/flyout_navigation';
 import { UniversalEntityFlyoutHeader } from './header';
 import { UniversalEntityFlyoutContent } from './content';
-import { FlyoutNavigation } from '../../shared/components/flyout_navigation';
+
+interface CommonError {
+  body: {
+    error: string;
+    message: string;
+    statusCode: number;
+  };
+}
+
+export const isCommonError = (error: unknown): error is CommonError => {
+  // @ts-ignore TS2339: Property body does not exist on type {}
```

Comment:
ignoring since this function is attempting to check if specific values exist on an unknown argument, there are ways around it but it becomes too messy and unreadable, worth the ignore imo